### PR TITLE
Allow use with rrapdg data package or RM/RME datasets

### PR DESCRIPTION
**Pull request by ConnectedSystems**
Tested changes with both rrap-dg data package and RME datasets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,30 +139,32 @@ This work was ported to Python from the original MATLAB developed by Dr. Veroniq
...
-(rrap-dg) $ rrapdg coral-cover downscale-icc [reefmod engine datapackage path] [target geopackage] [output path]
+(rrap-dg) $ rrapdg coral-cover downscale-icc [rrap-dg datapackage path] [target geopackage] [output path]
 ```
 
 For example, to downscale RME data for the Moore cluster defined by a geopackage:
 
 ```console
-(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/RME_v1 ./Moore.gpkg ./coral_cover.nc
+(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/rrapdg ./Moore.gpkg ./coral_cover.nc
+(rrap-dg) $ rrapdg coral-cover downscale-icc C:/example/rme_dataset ./Moore.gpkg ./coral_cover.nc
```
Comment (Zapiano) [inline README.md:167]: Does that also work with rme_dataset? Also, I think you need change `./Moore.gpkd` to `Moore`, no?

Comment (ConnectedSystems) [reply README.md:167]: Yes, it works with both the rrap-dg and RME darasets.

And, the idea now is to give it the geopackage file

Comment (Zapiano) [reply README.md:167]: Oh, sorry, I missed on file 🤦🏻‍♂️

File: rrap_dg/initial_coral_cover/icc.jl [hunk 2]
```diff
@@ -269,42 +326,32 @@ function downscale_icc(
...
+"""
+function downscale_icc(dataset::String, target_gpkg::String, output_path::String)::Nothing
+    gbr_gpkg = load_gbr_gpkg(dataset)
+    rme_icc = load_rme_cover(dataset, gbr_gpkg)
+    target_gpkg = GDF.read(target_gpkg)
+
+    icc = downscale_icc(rme_icc, gbr_gpkg, target_gpkg)
+
+    try
+        savecube(icc, output_path, driver=:netcdf)
```
Comment (Zapiano) [inline rrap_dg/initial_coral_cover/icc.jl:349]: For the cyclone datacube I'm using the flag `overwrite=true`. Do you prefer with or without it?

Comment (ConnectedSystems) [reply rrap_dg/initial_coral_cover/icc.jl:349]: I think without to avoid accidents...

File: rrap_dg/initial_coral_cover/icc.jl [hunk 3]
```diff
@@ -269,42 +326,32 @@ function downscale_icc(
...
-    savecube(icc, output_path, driver=:netcdf)
+# Arguments
+- `rme_dataset` : Path to rrap-dg data package
+- `target_gpkg` : Path to geopackage file defining area of interest
+- `output_path` : Path to export to resulting netCDF to
+"""
+function downscale_icc(dataset::String, target_gpkg::String, output_path::String)::Nothing
+    gbr_gpkg = load_gbr_gpkg(dataset)
+    rme_icc = load_rme_cover(dataset, gbr_gpkg)
+    target_gpkg = GDF.read(target_gpkg)
```
Comment (Zapiano) [inline rrap_dg/initial_coral_cover/icc.jl:344]: I tried running 
`rrapdg coral-cover downscale-icc C:/Users/pribeiro/AIMS/DataPackages/rrapdg/rrapdg_2024-02-14_rc ./Moore.gpkg ./coral_cover.nc` (like the README says) and got an error on this line (which makes sense).

Comment (ConnectedSystems) [reply rrap_dg/initial_coral_cover/icc.jl:344]: You supposed to give the path to the target geospatial data. 

Do you have a copy of `Moore.gpkg` in the current directory?

**Review by Zapiano — APPROVED** [review]
lgtm

**Pull request merged**